### DISCUSSION TITLE
Import additional PBF files after initial one

### DISCUSSION THREAD

**Discussion by ghost** [mediagis/nominatim-docker#441] [General]
Hello Folks! How you doing!?

I've been trying to run nominatim on my machine, using docker, and I've been facing some problems, when I try to add new PBF files to nominatim, after the initial import.

The documentation is not that clear about that, but gives the idea that this is possible. Well.

I have a docker container running with two volumes, one for pbf files, e another for postgres data. After the first import I try to run the following command to import the additional data:
```sudo -E -u nominatim nominatim import --osm-file /nominatim/data/centro-oeste-latest.osm.pbf```

But I receive the following output:

```
2023-03-30 19:52:31: Using project directory: /nominatim
2023-03-30 19:52:33: Creating database
createdb: error: database creation failed: ERROR:  database "nominatim" already exists
```

I know the database has already been created, but my question is. Is it possible to import new data after first import without having to re-built docker container? And if yes, what am I doing wrong!?

Thank you before hand!!!

**Comment by gsg-git** [comment]
Hello!
If you already have imported a file and want to add another one, you can use the add-data function to import the additional data as follows:

nominatim add-data --file <FILE>
nominatim refresh --postcodes
nominatim index -j <NUMBER OF THREADS>


From [here!](https://nominatim.org/release-docs/latest/admin/Advanced-Installations/)
